__tests__/warnings.spec.ts

```diff
@@ -135,6 +136,7 @@ describe('warnings', () => {
 
     await expect(router.push('//not-valid')).resolves.toBe(undefined)
     expect('cannot start with multiple slashes').toHaveBeenWarned()
+    expect(`No route matched`).toHaveBeenWarned()
```

Comment:
same as above, let's add a route `//not-valid`, although it doesn't make much sense

Reply:
not adding the route `//not-valid`, because the matcher won't be able to match it, I think if we add it will confuse maintainers in the future.